app/models/project.rb

```diff
@@ -1,4 +1,6 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
```

Comment:
Nice job setting up the associations. 

In here is where you would build a method to get your contestant count. (see my comment in your project_controller)